polaris-core/src/main/java/io/polaris/core/persistence/MetaStoreManagerFactory.java

```diff
@@ -42,5 +42,5 @@ public interface MetaStoreManagerFactory extends Discoverable {
 
   void setMetricRegistry(PolarisMetricRegistry metricRegistry);
 
-  Map<String, PolarisMetaStoreManager.PrincipalSecretsResult> bootstrapRealms(List<String> realms);
```

Comment:
May not be an issue since this is a relatively early time in the project but we may want to get in the habit of being careful when changing method signatures. This is a public interface so we probably would want to keep
```java
bootstrapRealms(List<String> realms) {
   bootstrapRealms(realms, false)
}
```

To avoid breaking implementers

Reply:
This is package-private and not public, right?

Reply:
It's `public` (Java interface, not a Java abstract class).